js/viz/export.d.ts

```diff
@@ -1,11 +1,24 @@
-import DOMComponent from '../core/dom_component';
-
 import {
     ExportFormat,
     HorizontalAlignment,
     VerticalAlignment,
 } from '../common';
 
+import dxChart from './chart';
+import dxPieChart from './pie_chart';
+import dxFunnel from './funnel';
+import dxSankey from './sankey';
+import dxTreeMap from './tree_map';
+import dxVectorMap from './vector_map';
+import dxSparkline from './sparkline';
+import dxBullet from './bullet';
+import dxBarGauge from './bar_gauge';
```

Comment:
Use `import type` to remove imports in run-time:

```suggestion
import type dxChart from './chart';
import type dxPieChart from './pie_chart';
import type dxFunnel from './funnel';
import type dxSankey from './sankey';
import type dxTreeMap from './tree_map';
import type dxVectorMap from './vector_map';
import type dxSparkline from './sparkline';
import type dxBullet from './bullet';
import type dxBarGauge from './bar_gauge';
```